website/src/pages/index.vue

```diff
@@ -2,104 +2,30 @@
   <Navbar />
   <chakra.main
     :min-h="['auto', 'auto', '100vh']"
-    w="100vw"
-    :px="[4, 10, 12]"
+    w="100%"
     pt="60px"
+    bg="vue.50"
```

Comment:
**nitpick:** I think the green background tends to make the page feel a bit busy and like it overflows because it's not balanced. I thought the hero looked better and more solid without the shadow and the green background.

You can compare the first and second image for contrast:
**With BG**
![image](https://user-images.githubusercontent.com/21237954/112363172-297d4200-8d10-11eb-8816-6ab2b15cfece.png)

**No BG**
![image](https://user-images.githubusercontent.com/21237954/112363456-79f49f80-8d10-11eb-9caf-3ca084ad094c.png)